src/dll/fuse/fuse_intf.c

```diff
@@ -1006,6 +1006,16 @@ static NTSTATUS fsp_fuse_intf_Open(FSP_FILE_SYSTEM *FileSystem,
         break;
     }
 
+    if (GrantedAccess & FILE_APPEND_DATA)
+    {
+        if (fi.flags == 0)
+        {
+            fi.flags = 1; // Need O_WRONLY as a bare minimum in order to append
+        }
+
+        fi.flags |= 8; /*O_APPEND*/
```

Comment:
This looks correct as the value of `O_APPEND` is 8 on both Windows and Cygwin.

Reply:
Yeah, I'm not a huge fan of using bare numbers here but it matches what was done above for setting O_RDONLY, O_WRONLY, and O_RDWR.

Reply:
Sorry, I had to do it like this as some headers of Windows (and definitely those of Cygwin) were off limits to this project as it does not use the CRT.